test/jdk/java/awt/Mixing/AWT_Mixing/MixingPanelsResizing.java

```diff
@@ -157,8 +167,10 @@ public void run() {
 
         SwingUtilities.invokeAndWait(new Runnable() {
             public void run() {
+
                 lLoc = frame.getLocationOnScreen();
```

Comment:
Remove this redundant blank line…

However, many `Runnable`s have a blank line right after the `run()` method declaration. In this case, I'd follow this style for consistency.